build.gradle

```diff
@@ -75,6 +78,10 @@ uploadArchives {
                 authentication(userName: project.findProperty("sonatypeUsername"), password: project.findProperty("sonatypePassword"))
             }
 
+            snapshotRepository(url: "https://broadinstitute.jfrog.io/broadinstitute/libs-snapshot-local/") {
+                authentication(userName: System.env.ARTIFACTORY_USERNAME, password: System.env.ARTIFACTORY_PASSWORD)
```

Comment:
@lbergelson Don't you also need the corresponding travis.yml entries for this ?

Reply:
Oh, are you just going to do this manually.

Reply:
just to be able to do it manually.  I should add travis as well, but that's a different pr I think.

Reply:
Oh wow - I see now. There is no travis CI at all for this repo.